src/nyc_trees/apps/mail/templates/mail/reservation_reminder.txt

```diff
@@ -0,0 +1,4 @@
+The following blockface reservations for user '{{ user.username }}' will expire soon:
```

Comment:
We will get "real" copy from Parks, but we will most definitely want to include an absolute link to the reservations page in the reminder to the mapper can act on it right away.